### ISSUE TITLE
Unused classes found when classifier is used in composes

### ISSUE THREAD

**Issue by jurosh** [atfzl/eslint-plugin-css-modules#4]
Example css:
```
.box{ ... }
.redBox {
 composes: box;
 background-color: red;
}
```

Example react code:
```
<div className={css.redBox}></div>
```

Now plugin in recommended configuration will say that `.box ` is unused.

Error message: `Unused classes found: box` & rule `css-modules/no-unused-class`

Plugin should probably care if classifier is used in another compose and do not throw error when it's actually used by another used class.

**Comment by atfzl** [comment]
You have a valid use case.
Will work on it this weekend.

**Comment by guywald1** [comment]
+1

**Comment by atfzl** [comment]
Added support for `composes` in 2.4.0.

I have also added support for `@extend` along with `composes`.
